isis/scripts/isis3VarInit.py

```diff
@@ -5,31 +5,68 @@
 import sys
 
 ##########################################################################################################
-#   Author:  Tyler Wilson
-#   Date  :  2018-10-05
-#   Description:  This script sets ISISROOT/ISIS3DATA/ISIS3TESTDATA for the user and is executed
-#   within the conda environment created for the ISIS3 installation.
-#   The data directory and test directory are optional command line arguments.  If the user chooses
-#   not to set them, they will both be placed created on the same level as the $ISISROOT directory
-#   within the conda environment.
+#
+#  This work is free of known copyright restrictions.  USGS-authored or produced data, 
+#  information, and software are in the public domain.
+#
+#
+#   Description:  This program builds the shell scripts that define the 
+#       ISISROOT/ISIS3DATA/ISIS3TESTDATA environment variables for the user 
+#       when the ISIS3 conda environment is activated, and clean up when it is
+#       deactivated.
+#
+#       The data directory and test directory are optional command line arguments.  
+#       If the user chooses not to set them, they will both be created in the
+#       $ISISROOT directory.
+#
 #   History:
-#       Author:  Tyler Wilson
+#       Author:  Tyler Wilson, USGS
+#       Date:    2018-10-05
+#       Description: Initial commit.
+#
+#       Author:  Tyler Wilson, USGS
 #       Date:    2018-11-01
 #       Description:  Removed a pair of lines which were causing output errors on Mac OS X and were not
 #                     required anyway.
 #
+#       Author:  Ross Beyer
+#       Date:    2018-11-
+#       Description: Streamlined the program, improved documentation, and made the directory and
+#                    file creation more `pythonic' rather than using system calls.
+#       To the extent possible under law, Ross Beyer has waived all copyright and related or 
+#       neighboring rights to his contribution to this file.  This work is published from
+#       the United States.
```

Comment:
Is this not covered by our [license](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/license.txt)? If it is not, then we need to update our license. @jlaura

Reply:
I don't think it is.  So let's resolve #588 and #591, and then I'll do that.

Reply:
So I used the unlicense model in a recent commit.